### ISSUE TITLE
TypeError: stateObj is undefined showing up in logs repeatedly

### ISSUE THREAD

**Issue by mdegat01** [finity69x2/binary-control-button-row#1]
I am seeing this log entry over and over again in my HA logs:
> 2020-04-11 09:57:59 ERROR (MainThread) [frontend.js.latest.202004072] {HA Address}/hacsfiles/binary-control-button-row/binary-control-button-row.js:136:4 TypeError: stateObj is undefined
> 2020-04-11 09:57:59 ERROR (MainThread) [frontend.js.latest.202004072] {HA Address}/hacsfiles/binary-control-button-row/binary-control-button-row.js:136:4 TypeError: stateObj is undefined

I replaced my HA address with `{HA Address}` but otherwise that's the message. My home assistant version is 0.108.3 and my `binary-control-button-row` version is 1.1. I don't see anything in the console logs that seems to be coming from this component, only the HA logs.

**Comment by finity69x2** [comment]
can you post the lovelace configuration for the entity row you are using the button with, please?

I'm on V108.3 and I just installed it on the test HA I use and it's working fine for me.

**Comment by mdegat01** [comment]
So the config I use it in is this:
```yaml
- entities:
    - entities:
        - light.kitchen
        - light.dining_room
        - entities:
            - entity: light.family_room_main
              name: Main
            - entity: light.family_room_front
              name: Front
          group_config:
            HiBrightness: 250
            LowBrightness: 5
            type: 'custom:light-brightness-preset-row'
          head:
            HiBrightness: 250
            LowBrightness: 5
            entity: light.family_room
            type: 'custom:light-brightness-preset-row'
          type: 'custom:fold-entity-row'
      group_config:
        HiBrightness: 250
        LowBrightness: 5
        type: 'custom:light-brightness-preset-row'
      head:
        HiBrightness: 250
        LowBrightness: 5
        entity: light.downstairs
        type: 'custom:light-brightness-preset-row'
      type: 'custom:fold-entity-row'
    - entities:
        - light.stairs
        - light.upstairs_hallway
        - light.bedroom
        - light.guest_room
      group_config:
        HiBrightness: 250
        LowBrightness: 5
        type: 'custom:light-brightness-preset-row'
      head:
        HiBrightness: 250
        LowBrightness: 5
        entity: light.upstairs
        name: Upstairs
        type: 'custom:light-brightness-preset-row'
      type: 'custom:fold-entity-row'
    - entities:
        - switch.deck_lights
        - switch.porch_light
        - switch.back_garden_zone
      group_config:
        type: 'custom:binary-control-button-row'
      head:
        entity: group.lights_outside
        icon: 'mdi:light-switch'
        name: Outside
        type: 'custom:binary-control-button-row'
      type: 'custom:fold-entity-row'
    - entities:
        - switch.pensieve_outlet
        - switch.burrow_outlet
        - switch.bedroom_tv_outlet
      group_config:
        type: 'custom:binary-control-button-row'
      head:
        entity: group.all_outlets
        icon: 'mdi:power-socket-us'
        name: Outlets
        type: 'custom:binary-control-button-row'
      type: 'custom:fold-entity-row'
    - entities:
        - switch.downstairs_presence_lights
        - switch.sunset_watcher
        - entities:
            - switch.coat_closet_light
            - switch.laundry_closet_light
            - switch.bedroom_closet_light
          group_config:
            type: 'custom:binary-control-button-row'
          head:
            entity: sun.sun
            icon: 'mdi:light-switch'
            name: Closet lights
            type: 'custom:dummy-entity-row'
          type: 'custom:fold-entity-row'
      group_config:
        type: 'custom:binary-control-button-row'
      head:
        entity: sun.sun
        icon: 'mdi:home-assistant'
        name: Others
        type: 'custom:dummy-entity-row'
      type: 'custom:fold-entity-row'
  show_header_toggle: false
  title: 'Lights, Outlets, and Switches'
  type: entities
```
However I have to note that since I reported this I've restarted a few times and haven't seen it again so I can't really reproduce it anymore either. If you also can't reproduce it then closing it is probably the right call.  If it comes up again I'll update.  Sorry about that.

**Comment by mdegat01** [comment]
@finity69x2 This has started appearing for me pretty much every restart again in recent releases. The message is identical to my original post hence why I'm replying here. Also my UI configuration actually hasn't significantly changed from what I showed above, I think I added a few more switches but that's basically still my main control panel on the primary dashboard.

However one thing of note I'm seeing this time. I use a different URL for accessing HA from my mobile apps then I do from my browser and I'm noticing that the URL appearing in these logs is always the one I use for my mobile apps. I use the android and mac companion apps, I'm wondering if this error has something to do with that. 

The component actually works fine everywhere for reference. I use it all the time and despite this startup error I've never had it not work when I go to use it. It just seems to be some error either as HA is coming back online or immediately after a it starts back up.
